src/sql/workbench/common/actions.contribution.ts

```diff
@@ -37,3 +37,16 @@ Registry.as<IConfigurationRegistry>(ConfigExtensions.Configuration).registerConf
 		}
 	}
 });
+
+Registry.as<IConfigurationRegistry>(ConfigExtensions.Configuration).registerConfiguration({
```

Comment:
why is this necessary? The connect dialog only gets shown on startup if there are no servers, generally first launch. I'm not sure of a case where we wouldn't want to show it

Reply:
this is also for test environment setup, most test cases will start with no connection history (new temp user data folder). 1. for extension test (not UI automation based), there is no way for us to close the dialog programatically.
2. for ui automation, some test may not need this dialog to be on, and they have to close it, rather that having each test needing to worry about this, I would prefer ADS to start without this dialog opened.